lighthouse-core/gather/gatherers/cache-contents.js

```diff
@@ -46,12 +46,7 @@ class CacheContents extends Gatherer {
   async afterPass(passContext) {
     const driver = passContext.driver;
 
-    /** @type {Array<string>|void} */
-    const cacheUrls = await driver.evaluateAsync(`(${getCacheContents.toString()}())`);
-    if (!cacheUrls || !Array.isArray(cacheUrls)) {
```

Comment:
Extra defensive coding, or am I missing something?

Reply:
In JS evaluating arrays for truthiness doesn't consider array length. A length check does happen in python, I believe.